files/image_config/topology/topology.sh

```diff
@@ -7,17 +7,20 @@
 
 start() {
     TOPOLOGY_SCRIPT="topology.sh"
-    PLATFORM=`sonic-cfggen -H -v DEVICE_METADATA.localhost.platform`
-    HWSKU=`sonic-cfggen -d -v 'DEVICE_METADATA["localhost"]["hwsku"]'`
+    PLATFORM=${PLATFORM:-`sonic-cfggen -H -v DEVICE_METADATA.localhost.platform`}
+    HWSKU=${HWSKU:-`sonic-cfggen -d -v 'DEVICE_METADATA["localhost"]["hwsku"]'`}
     /usr/share/sonic/device/$PLATFORM/$HWSKU/$TOPOLOGY_SCRIPT start
 }
 stop() {
     TOPOLOGY_SCRIPT="topology.sh"
-    PLATFORM=`sonic-cfggen -H -v DEVICE_METADATA.localhost.platform`
-    HWSKU=`sonic-cfggen -d -v 'DEVICE_METADATA["localhost"]["hwsku"]'`
+    PLATFORM=${PLATFORM:-`sonic-cfggen -H -v DEVICE_METADATA.localhost.platform`}
+    HWSKU=${HWSKU:-`sonic-cfggen -d -v 'DEVICE_METADATA["localhost"]["hwsku"]'`}
```

Comment:
Suggest using the same syntax for these variables. Either change former to `'DEVICE_METADATA["localhost"]["platform"]'` or change latter to `DEVICE_METADATA.localhost.hwsku`